### DISCUSSION TITLE
New postgresql and sqlite images?

### DISCUSSION THREAD

**Discussion by tracymiranda** [wolfi-dev/os#68] [Q&A] [answered]
I saw this tweet https://twitter.com/lorenc_dan/status/1587408701022953473?s=20&t=c7VvuMk9JgpwYS3Zu9Z-Ew and was curious about the sizes of the new postgresql and sqlite images?

**Comment by imjasonh** [comment, marked as answer]
There's a postgres Chainguard Image that includes the postgres Wolfi package, here: https://github.com/chainguard-images/postgres -- it's 13 MB (vs Docker's [`postgres`](https://hub.docker.com/_/postgres), 133MB)

There isn't currently an image that includes the `sqlite` Wolfi package.
